app/main.py

```diff
@@ -5,7 +5,7 @@
 app = FastAPI()
 
 # Mount the static files directory
-app.mount("/static", StaticFiles(directory="templates/static"), name="static")
+app.mount("/static", StaticFiles(directory="app/templates/static"), name="static")
```

Comment:
Move image_manipulation.js so that this doesn't break

Reply:
Thank you for the feedback. I'll move `image_manipulation.js` to the `app/templates/static/` directory to ensure it doesn't break. I'll make the necessary changes.

Reply:
Also move the html and css

Reply:
Thank you for the feedback. I'll move the HTML and CSS files to the `app/templates/static/` directory to ensure everything is correctly located and accessible. I'll make the necessary changes.